src/Core/Components/Rating/FluentRating.razor.cs

```diff
@@ -32,6 +32,11 @@ public partial class FluentRating : FluentInputBase<int>
     /// </summary>
     [Parameter] public Icon IconEmpty { get; set; } = new CoreIcons.Regular.Size20.Star();
 
+    /// <summary>
+    /// Fires when hovered value changes. Value will be null if no rating item is hovered.
+    /// </summary>
+    [Parameter] public EventCallback<int?> HoveredValueChanged { get; set; }
```

Comment:
I wouldn't use this event. If the user wants to display a label, it shouldn't be at the MouseOver but after selecting a value. Making a component too complex risks confusing it :-)

For example, in the case of this event, what about mobiles (and Touch events) and what about accessibility?

Reply:
I don't use mouseover but onpointerover. I use it to manage the icon change and then I invoke HoveredValueChanged